aiidalab_widgets_base/structures.py

```diff
@@ -658,8 +660,8 @@ def search(self, _=None):
         matches = {n[0] for n in qbuild.iterall()}
         matches = sorted(matches, reverse=True, key=lambda n: n.ctime)
 
-        options = OrderedDict()
-        options[f"Select a Structure ({len(matches)} found)"] = False
+        options = []
```

Comment:
Since `ipywidget` use tuple as default for `opitions` (see https://github.com/jupyter-widgets/ipywidgets/blob/52663ac472c38ba12575dfb4979fa2d250e79bc3/python/ipywidgets/ipywidgets/widgets/widget_selection.py#L156-L166)

Reply:
See the comment I just added, I think it is better to initiate the `options` as a tuple.